core/ledger/genesis/genesis.go

```diff
@@ -55,26 +55,11 @@ func MakeGenesis() error {
 
 		defer func() {
 			if !genesisBlockExists && makeGenesisError == nil {
-				genesisLogger.Infof("Adding %d system chaincodes to the genesis block.", len(genesisTransactions))
+				genesisLogger.Info("Adding %d system chaincodes to the genesis block.", len(genesisTransactions))
```

Comment:
Why was this changed from `Infof` to `Info`? Same question for the other logging changes in this file.